libs/design/src/molecules/paginator/paginator.component.html

```diff
@@ -0,0 +1,14 @@
+<span (click)="onNotifyPrevPageChange()" *ngIf="showPrevPageChange()" class="daff-paginator__prev daff-paginator__element">Prev</span>
```

Comment:
`previous` and `next` should have its own style, independent of `element/page-item`

```
display: flex;
margin-right: 20px; / margin-left: 20px; for next
align-items: center;
```

Reply:
done